reel-picker/src/main/java/com/innowisegroup/reelpicker/picker/ui/TimePickerFragment.kt

```diff
@@ -24,13 +20,10 @@ internal class TimePickerFragment : Fragment() {
     private var hours: CustomNumberPicker? = null
     private var minutes: CustomNumberPicker? = null
```

Comment:
Let's make it non-null since we garantee that we defenetly initialize it in onCreateView method and there will be no cases where we get null. 

private lateinit var hours: CustomNumberPicker
private lateinit var minutes: CustomNumberPicker